eo-maven-plugin/pom.xml

```diff
@@ -132,6 +135,17 @@ SOFTWARE.
       <version>${runtime.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.github.hse-eolang</groupId>
+      <artifactId>transpiler</artifactId>
+      <version>1.0.5</version>
+    </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-core</artifactId>
+      <version>1.3</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
@nlchar this is redundant, we have it coming from the parent. Just delete it here, I'm sure everything will work just fine.